[2025-06-30T06:03:07.662+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-06-30T06:03:07.891+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: mysql_dag.insert_data manual__2025-06-30T06:03:06.078659+00:00 [queued]>
[2025-06-30T06:03:07.902+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: mysql_dag.insert_data manual__2025-06-30T06:03:06.078659+00:00 [queued]>
[2025-06-30T06:03:07.903+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-06-30T06:03:07.964+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): insert_data> on 2025-06-30 06:03:06.078659+00:00
[2025-06-30T06:03:07.975+0000] {standard_task_runner.py:63} INFO - Started process 1108 to run task
[2025-06-30T06:03:07.979+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'mysql_dag', 'insert_data', 'manual__2025-06-30T06:03:06.078659+00:00', '--job-id', '61', '--raw', '--subdir', 'DAGS_FOLDER/mysql.py', '--cfg-path', '/tmp/tmpk9cptim9']
[2025-06-30T06:03:07.986+0000] {standard_task_runner.py:91} INFO - Job 61: Subtask insert_data
[2025-06-30T06:03:08.013+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.10/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-06-30T06:03:08.081+0000] {task_command.py:426} INFO - Running <TaskInstance: mysql_dag.insert_data manual__2025-06-30T06:03:06.078659+00:00 [running]> on host 76cdd04f4df6
[2025-06-30T06:03:08.240+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Karthik' AIRFLOW_CTX_DAG_ID='mysql_dag' AIRFLOW_CTX_TASK_ID='insert_data' AIRFLOW_CTX_EXECUTION_DATE='2025-06-30T06:03:06.078659+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-30T06:03:06.078659+00:00'
[2025-06-30T06:03:08.243+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-06-30T06:03:09.953+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-06-30T06:03:09.956+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-06-30T06:03:09.977+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=mysql_dag, task_id=insert_data, run_id=manual__2025-06-30T06:03:06.078659+00:00, execution_date=20250630T060306, start_date=20250630T060307, end_date=20250630T060309
[2025-06-30T06:03:10.048+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-06-30T06:03:10.086+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-06-30T06:03:10.093+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
